### PR TITLE
[Merged by Bors] - feat: port Analysis.Calculus.Deriv.Inv

### PR DESCRIPTION
**Pull request by urkud**
---

This file is going to substantially change in `mathlib3` very soon.

- [x] depends on: #4436
- [x] depends on: #4440

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by semorrison** [comment]
This PR/issue depends on:
* leanprover-community/mathlib4#4436
* leanprover-community/mathlib4#4440
By **[Dependent Issues](https://github.com/z0al/dependent-issues)** (🤖). Happy coding!

File: Mathlib/Analysis/Calculus/Deriv/Inv.lean [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+import Mathlib.Analysis.Calculus.Deriv.Comp
+
+/-!
+# Derivatives of `x ↦ x⁻¹` and `f x / g x`
+
+In this file we prove `(x⁻¹)' = -1 / x ^ 2`, `((f x)⁻¹)' = -f' x / (f x) ^ 2`, and
+`(f x / g x)' = (f' x * g x - f x * g' x) / (g x) ^ 2` for different notions of derivative.
+
+For a more detailed overview of one-dimensional derivatives in mathlib, see the module docstring of
+`analysis/calculus/deriv/basic`.
```
Comment (fpvandoorn) [inline Mathlib/Analysis/Calculus/Deriv/Inv.lean:21]:
```suggestion
`Analysis/Calculus/Deriv/Basic`.
```
bors d+

**Comment by semorrison** [comment]
This PR/issue depends on:
* ~~leanprover-community/mathlib4#4436~~
* ~~leanprover-community/mathlib4#4440~~
By **[Dependent Issues](https://github.com/z0al/dependent-issues)** (🤖). Happy coding!

**Comment by Parcly-Taxel** [comment]
Is it going to substantially change in mathlib3? Where is the PR that will effect that change?

**Comment by ChrisHughes24** [comment]
bors merge

**Pull request closed without merging**